.github/workflows/codestyle.yml

```
@@ -3,15 +3,19 @@ name: Basic tests
 on: [push, pull_request]
 
 jobs:
-  eslint:
+  pre-commit:
     runs-on: ubuntu-latest
+    if: github.event_name != 'push' || github.repository == 'DIRACGrid/DIRAC'
```

I don't think it will run on `DIRACGrid/DIRAC`
```suggestion
    if: github.event_name != 'push' || github.repository == 'DIRACGrid/WebAppDIRAC'
```